orga/tests/integration/components/sidebar-menu-test.js

```diff
@@ -90,4 +90,27 @@ module('Integration | Component | sidebar-menu', function(hooks) {
     // then
     assert.dom('.sidebar-menu__documentation-item').doesNotExist();
   });
+
+  test('it should display Certifications menu in the sidebar-menu', async function(assert) {
```

Comment:
```suggestion
  test('it should display Certifications menu in the sidebar-menu for a sco organization that is managing students when the appropriate feature toggle is enabled', async function(assert) {
```
peut-être pas ce remplacement là précisément, mais il manque du contexte dans l'intitulé du test à mon sens